src/setupPagination.js

```diff
@@ -0,0 +1,52 @@
+import updateData from "./updateData.js";
+
+export default setupPagination;
+
+function updateSelectedButton(currentPage) {
+    const oldButtons = document.getElementsByClassName("selected");
+    Array.from(oldButtons).forEach(button => button.classList.remove("selected"));
+
+    const newButton = document.getElementsByClassName(currentPage);
+    Array.from(newButton).forEach(button => button.classList.add("selected"));
+}
+
+function goToNextPage(currentPage, maxPage) {
+    return function() {
+        if (currentPage !== maxPage) {
+            currentPage += 1;
+            updateData(currentPage);
+            updateSelectedButton(currentPage);
+        }
+    }
+}
```

Comment:
Why are you returning a function here? I don't get it.

Reply:
I was having a problem with the `.onclick` method. The functions that were passed to it were automatically being called.

It is the same problem as the one from this post: https://stackoverflow.com/questions/16310423/addeventlistener-calls-the-function-without-me-even-asking-it-to

I applied the answer that seemed the most cleaner to read. But in our pair up meeting Rafael told me a better way that uses arrow function to solve this.